lib/date_time/date_time.c

```diff
@@ -63,9 +70,11 @@ static struct sntp_time sntp_time;
 
 K_SEM_DEFINE(time_fetch_sem, 0, 1);
 
-static struct k_work_delayable time_work;
+static void date_time_handler(struct k_work *work);
+K_WORK_DELAYABLE_DEFINE(time_work, date_time_handler);
 
 static struct time_aux {
+	/* TODO: Should we remove date_time_utc and rely on OS time? */
```

Comment:
So, is there conclusion for this TODO? It's a bit confusing to see TODO in a form of a question.